src/controllers/error.controller.ts

```diff
@@ -3,7 +3,13 @@ import { logger } from "../utils/logger";
 import * as config from "../config";
 
 const pageNotFound = (req: Request, res: Response) => {
-  return res.status(404).render(config.NOT_FOUND_PAGE);
+  let isRegistration = true;
```

Comment:
const isRegistration : boolean = req.path.startsWith(config.UPDATE_LANDING_URL);

Reply:
@AndreKramer7654 , can change it `let isRegistration : boolean = req.path.startsWith(config.UPDATE_LANDING_URL);` as I change the value later and with a const its not possible